kubernetes/yamls/development/elasticsearch/es-deployment.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: es
+  namespace: elasticsearch
+  labels:
+    component: elasticsearch
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: elasticsearch
+    spec:
+      serviceAccount: elasticsearch
+      containers:
+      - name: es
+        livenessProbe:
+          httpGet:
+            path: /
```

Comment:
@singhpratyush can you use https://www.elastic.co/guide/en/elasticsearch/reference/current/cluster-health.html instead of just a `/` ?